### ISSUE TITLE
"ePub is not defined" error in console.

### ISSUE THREAD

**Issue by scallemang** [futurepress/epub.js#766]
Howdy,

Trying to get up and running with a WP proof-of-concept using the Sage theme.

Added `epubjs` with yarn. My JS looks like this:

```
import ePub from 'epubjs/dist/epub.min';

export default {
  init() {
    // JavaScript to be fired on all pages
  },
  finalize() {
    // JavaScript to be fired on all pages, after page specific JS is fired
    $(document).ready(function(){
      if($('div.ebook')) {
        var book = ePub( $('div.ebook').attr('data-ebook') );
        /* eslint-disable */
        console.log(book);
        /* eslint-enable */
        var rendition = book.renderTo("ebook-reader", {width: 600, height: 400});
        rendition.display();
      }
    });
  },
};
```

The page where I'm testing it is here [http://srh.scallemang.ca/sample-page/](http://srh.scallemang.ca/sample-page/). Any advice on what I'm doing incorrectly here?

Thanks so much.

**Comment by fchasen** [comment]
Could you try `import ePub from 'epubjs'; `?

**Comment by scallemang** [comment]
@fchasen Happy to, Fred.

I ran the `yarn build` command and am getting this error after moving files to my server:

```
popper.js:2435 Uncaught ReferenceError: ePub is not defined
    at new Contents (popper.js:2435)
    at IframeView.onLoad (main.js:15293)
    at IframeView.<anonymous> (main.js:15261)
```

However, it's trying to load that Popper file from my computer (`webpack:////Users[...]`), so I tried to run the `build:production` command. This fails to compile, with the error: `scripts/main_e90a5adc.js from UglifyJs \n Unexpected token: name (f) [scripts/main_e90a5adc.js:664,5];`.

Open to any suggestions, this here is well over my head. Thanks so much!

**Comment by jopit** [comment]
I'm seeing something similar.  Adding `global.ePub = require('epubjs').default;` at the start of my code got it working.

More details:

I'm using version 0.3.63.

If I use `const ePub = require('epubjs');` I get an error when I call `ePub('/path/to/file.epub')` in my code:
```
Uncaught TypeError: ePub is not a function
    at showBook (book.js:22)
    at EventEmitter.ipcRenderer.on (book.js:18)
    at emitTwo (events.js:126)
    at EventEmitter.emit (events.js:214)
```

Changing the require to `const ePub = require('epubjs').default;` gets past that error, but then I run into the `ePub is not defined` error:
```
Uncaught ReferenceError: ePub is not defined
    at new Contents (my-app/node_modules/epubjs/lib/contents.js:68)
    at IframeView.onLoad (my-app/node_modules/epubjs/lib/managers/views/iframe.js:409)
    at IframeView.<anonymous> (my-app/node_modules/epubjs/lib/managers/views/iframe.js:377)
```
Adding the `ePub` function to the global object then gets around that problem.

Seem like something's wrong with what the `epubjs` module exports.

**Comment by fchasen** [comment]
There was a stray global variable for the current `ePub.version` in Content, fixed in `epubjs@0.3.66`
